ocean-client/src/vault-maxi.ts

```diff
@@ -102,7 +119,7 @@ export async function main(event: maxiEvent, context: any): Promise<Object> {
       //do checkSetup after general checks, so that a successfully checkSetup without errors means its really all good.
       if (event) {
         if (event.checkSetup) {
-          let result = await program.doAndReportCheck(telegram)
+          let result = await program.doAndReportCheck(telegram, oceansToUse.slice(3))
```

Comment:
why slice 3?

Reply:
first 3 entries are the prefilled urls (defichain.io, defichain.com, defichain.io to alternate 2 times). don't want to send them in the report as custom fallback urls.